ground/openpilotgcs/src/plugins/usagetracker/usagetrackerplugin.cpp

```diff
@@ -81,11 +81,11 @@ void UsageTrackerPlugin::onAutopilotConnect()
             message.setIcon(QMessageBox::Information);
             message.addButton(tr("Yes, count me in"), QMessageBox::AcceptRole);
             message.addButton(tr("No, I will not help"), QMessageBox::RejectRole);
-            message.setText(tr("Openpilot GCS has a function to collect limited anonymous information about "
-                               "the usage of the application itself and the OpenPilot hardware connected to it.<p>"
+            message.setText(tr("%1 GCS has a function to collect limited anonymous information about "
```

Comment:
I think you can use both. (.arg() twice) Imagine we had 2 gcs, say "LibrePilot GCS" and "LibrePilot Copter Control GCS" that both used this code.